libs/ledger-live-common/src/families/solana/bridge/mock.ts

```diff
@@ -1,14 +1,22 @@
 import { makeLRUCache, minutes } from "@ledgerhq/live-network/cache";
 import { Message } from "@solana/web3.js";
 import { flow, isArray, isEqual, isObject, isUndefined, mapValues, omitBy } from "lodash/fp";
-import { ChainAPI, Config, cached, getChainAPI, logged, queued } from "../api";
-import { Functions } from "../utils";
-import { makeBridges } from "./bridge";
+import {
+  ChainAPI,
+  Config,
+  cached,
+  getChainAPI,
+  logged,
+  queued,
+} from "@ledgerhq/coin-solana/api/index";
+import { Functions } from "@ledgerhq/coin-solana/utils";
+import { makeBridges } from "@ledgerhq/coin-solana/bridge/bridge";
+import { SolanaAddress, SolanaSignature, SolanaSigner } from "@ledgerhq/coin-solana/signer";
 import { getMockedMethods } from "./mock-data";
 
 function mockChainAPI(config: Config): ChainAPI {
   const mockedMethods = getMockedMethods();
-  const api = new Proxy(
+  const api: any = new Proxy(
```

Comment:
Is it necessary ? The type should be inferered. Actually where is proxy coming from ?